### PR TITLE
Localisation updates from https://translatewiki.net.

### PR DESCRIPTION
**Pull request by translatewiki**
Translation updates

### CODE REVIEW

**Comment by adhiamboperes** [comment]
@seanlip,  we manually remove all the `fuzzy` attributes from the translations of the renamed strings but  @translatewiki forcepushed them again. Could we have someone in the translations team de-fuzzy them on translate wiki?

**Comment by seanlip** [comment]
Could you please contact Pretty about this? I will introduce you by email.

Thanks!

**Comment by adhiamboperes** [comment]
Updated **Pretty** on the expected actions, waiting for feedback from translations team.

**Comment by Nikerabbit** [comment]
Hi. Are you still having troubles with translation updates from our side? I noticed that this has been open for two months.

**Comment by adhiamboperes** [comment]
> Hi. Are you still having troubles with translation updates from our side? I noticed that this has been open for two months.

Hi @Nikerabbit, this PR hasn't been merged because we have translations that haven't been verified(fuzzy). We changed the  values of some strings(same string name, but different content) and translate wiki pushed the old translations, so our translation team needs to manually fix this on the wiki side.

File: app/src/main/res/values-ar/strings.xml
Comment (adhiamboperes) [inline app/src/main/res/values-ar/strings.xml]: These two look like empty strings.

File: app/src/main/res/values-pcm-rNG/strings.xml
Comment (adhiamboperes) [inline app/src/main/res/values-pcm-rNG/strings.xml]: This stringand the two below are broken too. In the immediate last force push, it was okay but now it's empty.

File: app/src/main/res/values-ar/strings.xml
Comment (adhiamboperes) [inline app/src/main/res/values-ar/strings.xml]: These four strings are empty now but they were not  previously.

File: app/src/main/res/values-pt-rBR/strings.xml
Comment (adhiamboperes) [inline app/src/main/res/values-pt-rBR/strings.xml]: Missing translations for this set

File: app/src/main/res/values-pt-rBR/strings.xml
Comment (adhiamboperes) [inline app/src/main/res/values-pt-rBR/strings.xml]: Empty string here too

File: app/src/main/res/values-sw/strings.xml
Comment (adhiamboperes) [inline app/src/main/res/values-sw/strings.xml]: Empty string

File: app/src/main/res/values-sw/strings.xml
Comment (adhiamboperes) [inline app/src/main/res/values-sw/strings.xml]: Empty strings

**Review by adhiamboperes — DISMISSED** [review]
Some of the strings are empty in the latest push, and I'm not sure why. Two things I've checked:
1. The base English strings are formatted correctly
2.  The previous force push had the correct version of the translations. Did something happen on translate wiki?

I think we might need to check the @translatewiki dashboard and maybe import the translations manually, or get to the root of the issue.

File: app/src/main/res/values-ar/strings.xml
Comment (Nikerabbit) [reply app/src/main/res/values-ar/strings.xml]: I looked up this one and it they got emptied on translatewiki.net when https://github.com/oppia/oppia-android/commit/30ba44dc91e42d0fd960aac9306f76170d3ba3e4#diff-275e22e5424874865d61be0a790c48168da5998ec6dd530c83fb33005bfa4449R524 was imported. My guess is that the unescaped html in the messages choke our parser which caused the strings to be empty. This is a bug on our side: we are using PHP's `SimpleXMLElement` which doesn't provide method to get innerHTML.

This is probably the cause for all other empty strings too

Comment (adhiamboperes) [reply app/src/main/res/values-ar/strings.xml]: > I looked up this one and it they got emptied on translatewiki.net when [30ba44d#diff-275e22e5424874865d61be0a790c48168da5998ec6dd530c83fb33005bfa4449R524](https://github.com/oppia/oppia-android/commit/30ba44dc91e42d0fd960aac9306f76170d3ba3e4#diff-275e22e5424874865d61be0a790c48168da5998ec6dd530c83fb33005bfa4449R524) was imported. My guess is that the unescaped html in the messages choke our parser which caused the strings to be empty. This is a bug on our side: we are using PHP's `SimpleXMLElement` which doesn't provide method to get innerHTML.
> 
> This is probably the cause for all other empty strings too

Thanks @Nikerabbit. Do you have a suggestion for how we can work around this at the moment?

Comment (Nikerabbit) [reply app/src/main/res/values-ar/strings.xml]: The values should re restored with escaping or directly in translatewiki.net

**Comment by BenHenning** [comment]
I believe we'll need to address #5357 before this can be merged to work around the Translatewiki import issue with CDATA strings.

**Comment by adhiamboperes** [comment]
@Nikerabbit, just updating that we're working on fixing the import issues causing empty translations per https://github.com/oppia/oppia-android/issues/5357. We generally expect a full solution for end of week next week, to be able to merge this PR.

**Comment by adhiamboperes** [comment]
In the latest push, the empty strings problem was not fixed. The next step is to do a manual fix either on translate wiki or update the strings in a seperate PR

File: app/src/main/res/values-ar/strings.xml
Comment (BenHenning) [reply app/src/main/res/values-ar/strings.xml]: @Nikerabbit thanks for the earlier insight. We've now submitted a change that should be correctly escaping HTML tag elements rather than using CDATA (see #5524), but it seems this issue is still happening. Does this mean we'll need to fix the strings on the Translatewiki side or is there still something problematic with our string formatting?

**Comment by BenHenning** [comment]
I left a follow-up comment @adhiamboperes seeing if we can get some thoughts from the Translatewiki team. Which strings would we need to update in Translatewiki at this point? It seems that some of them correctly repulled (i.e. now have 'fuzzy' set).

File: app/src/main/res/values-ar/strings.xml
Comment (Nikerabbit) [reply app/src/main/res/values-ar/strings.xml]: The old translations are available via the history of individual pages listed in
* https://translatewiki.net/w/i.php?title=Special:Translations&message=Oppia%3Aoppia-android-app-faq_answer_app_language
* https://translatewiki.net/w/i.php?title=Special:Translations&message=Oppia%3Aoppia-android-app-faq_answer_update_app

It seems that both `faq_answer_app_language` and `faq_answer_bug_reporting` have changed in their content, so maybe it would be better if I just deleted the broken translations?

Comment (adhiamboperes) [reply app/src/main/res/values-ar/strings.xml]: It might be okay to do that.

**Comment by Nikerabbit** [comment]
I've removed all the empty translations. Maybe this is good to go now?

**Comment by adhiamboperes** [comment]
> I've removed all the empty translations. Maybe this is good to go now?


Thanks @Nikerabbit! the PR is clean now.

@seanlip, @BenHenning, I am merging the PR now since the issues have been resolved. The Portuguese team has updated the problematic translations, Arabic is partially done but the rest are not in yet.

**Comment by adhiamboperes** [comment]
@BenHenning/ @seanlip, could one of you please dismiss my old review? It is outdated so I can't find and resolve the comments.

**Comment by BenHenning** [comment]
> @BenHenning/ @seanlip, could one of you please dismiss my old review? It is outdated so I can't find and resolve the comments.

Done @adhiamboperes!

**Review by adhiamboperes — APPROVED** [review]
Latest commits LGTM.

**Pull request merged**
